### DISCUSSION TITLE
Upgrade from GitHub enterprise 2.8 to 2.19

### DISCUSSION THREAD

**Discussion by robinson031** [community/community#21832] [Enterprise]
<p>Hello,</p>
<p>Is there any way if we can upgrade from GitHub enterprise 2.8 to latest 2.19 without loosing any data from the existing 2.8 version ?</p>
<p>Please let me know if any.</p>
<p>Thanks,</p>
<p>Robinson</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! Check out the support article on upgrading GitHub Enterprise versions here: <a href="https://help.github.com/en/enterprise/2.18/admin/installation/upgrading-github-enterprise-server" rel="nofollow noopener">https://help.github.com/en/enterprise/2.18/admin/installation/upgrading-github-enterprise-server</a></p>
